server/routes/userProfileRouter.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const router = express.Router()
+const {
+  profile,
+  getProfile,
+  getAllProfiles
+} = require('../controllers/userProfileController')
+
+router.patch('/:userId', profile)
+router.get('/get-profile/:userId', getProfile)
+router.get('/get-all_profiles/', getAllProfiles)
```

Comment:
Maybe a typo, you missed hyphenating the route name here :)

```suggestion
router.get('/get-all-profiles/', getAllProfiles)
```